### DISCUSSION TITLE
Adding Button to Individual Rows

### DISCUSSION THREAD

**Discussion by rogerad** [rappasoft/laravel-livewire-tables#365] [General]
hi 
 i am trying to add a button to each row in the report for edit , view ,delete the product in the model

is this possible and how do i do it ?
similar to picture or just 3 buttons which i can assign routes
<img width="374" alt="rapas buts" src="https://user-images.githubusercontent.com/18638656/122872512-e44ec780-d373-11eb-84d1-f8a30df4e87b.png">

**Comment by rogerad** [comment]
i  copied buttons from manually constructed livewire datatable without the package
BUTTONS NOW WORK  and go to  the appropriate route.

 **But the route is loaded at the time as the datatable.**

 how can i set the route to load only when the EDIT  button is clicked on the row ?

**Reply by rogerad** [reply]
all fixed - working now

**Comment by jihadismail8** [comment]
and how you did it ?
